src/platform/plugins/private/vis_default_editor/public/default_editor.tsx

```diff
@@ -22,11 +20,91 @@ import {
 import { KibanaContextProvider } from '@kbn/kibana-react-plugin/public';
 import { Storage } from '@kbn/kibana-utils-plugin/public';
 
+import { css } from '@emotion/react';
+import { useMemoCss } from '@kbn/css-utils/public/use_memo_css';
 import { DefaultEditorSideBar } from './components/sidebar';
 import { getInitialWidth } from './editor_size';
 
 const localStorage = new Storage(window.localStorage);
 
+const defaultEditorStyles = {
+  base: (euiThemeContext: UseEuiTheme) =>
+    css({
+      flex: '1 1 auto',
+      display: 'flex',
+      [euiBreakpoint(euiThemeContext, ['xs', 's', 'm'])]: {
+        flexDirection: 'column', // change the editor direction to column
+      },
+    }),
+  // Collapsible sidebar container
+  collapsibleSidebar: (euiThemeContext: UseEuiTheme) =>
+    css({
+      flexGrow: 1,
+      [euiBreakpoint(euiThemeContext, ['xs', 's', 'm'])]: {
+        width: '100% !important', // force the editor to take 100% width
+        flexGrow: 0,
+      },
+    }),
+  // !importants on width and height are required to override resizable panel inline widths
+  collapsibleSidebarIsClosed: (euiThemeContext: UseEuiTheme) =>
+    css({
+      minWidth: 0,
+      width: `${euiThemeContext.euiTheme.size.xl} !important`, // Just enough room for the collapse button
+      '.visEditorSidebar': {
+        display: 'none',
+        paddingLeft: 0,
+      },
+      [euiBreakpoint(euiThemeContext, ['xs', 's', 'm'])]: {
+        height: `${euiThemeContext.euiTheme.size.xxl} !important`, // Just enough room for the collapse button
+      },
+    }),
+  // Resizer
+  resizer: (euiThemeContext: UseEuiTheme) =>
+    css({
+      height: 'auto',
+      [euiBreakpoint(euiThemeContext, ['xs', 's', 'm'])]: {
+        display: 'none', // hide the resizer button
+      },
+    }),
+  resizerIsHidden: css({
+    display: 'none',
+  }),
+  // Canvas area
+  visWrapper: (euiThemeContext: UseEuiTheme) =>
+    css({
+      [euiBreakpoint(euiThemeContext, ['xs', 's', 'm'])]: {
+        width: '100% !important',
+        flex: 1,
+        display: 'flex',
+        flexDirection: 'column',
+      },
+    }),
+  visWrapperExpanded: css({ width: '100% !important' }),
+  visualization: css({
+    display: 'flex',
+    flex: '1 1 auto', // Fixes IE bug: the editor overflows a visualization on small screens
+    overflow: 'hidden',
+  }),
+  canvas: ({ euiTheme }: UseEuiTheme) =>
+    css({
+      backgroundColor: euiTheme.colors.emptyShade,
+      display: 'flex',
+      flexDirection: 'row',
+      overflow: 'auto',
+      flexShrink: 1,
+      flexBasis: '100%',
+
+      '&:embedded': {
```

Comment:
I think `&:` is not used for classes, so I guess this style won't be applied. However, I see that this style (`flexShrink: 1, flexBasis: '100%'`) is kinda duplicate so I guess we can remove it
```suggestion
      '&.embedded': {
```

Reply:
Nice catch! removed them completely. Thank you!